tskv/src/memcache.rs

```diff
@@ -281,11 +283,67 @@ impl RowData {
     }
 }
 
+impl PartialOrd for RowData {
+    fn partial_cmp(&self, other: &Self) -> Option<cmp::Ordering> {
+        Some(self.ts.cmp(&other.ts))
+    }
+}
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct OrderedRowsData {
+    rows: OrderedSkipList<RowData>,
+}
+
+impl OrderedRowsData {
+    pub fn new() -> Self {
+        let mut rows: OrderedSkipList<RowData> = OrderedSkipList::new();
+        unsafe { rows.sort_by(|a: &RowData, b: &RowData| b.partial_cmp(a).unwrap()) }
+        Self { rows }
+    }
+
+    pub fn get_rows(self) -> OrderedSkipList<RowData> {
+        self.rows
+    }
+
+    pub fn get_ref_rows(&self) -> &OrderedSkipList<RowData> {
+        &self.rows
+    }
+
+    pub fn clear(&mut self) {
+        self.rows.clear()
+    }
+
+    pub fn insert(&mut self, row: RowData) {
+        self.rows.insert(row);
+    }
+
+    pub fn retain(&mut self, mut f: impl FnMut(&RowData) -> bool) {
+        self.rows.retain(|row| f(row));
+    }
+}
+
+impl Default for OrderedRowsData {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl Clone for OrderedRowsData {
+    fn clone(&self) -> Self {
+        let mut clone_rows: OrderedSkipList<RowData> = OrderedSkipList::new();
+        unsafe { clone_rows.sort_by(|a: &RowData, b: &RowData| b.partial_cmp(a).unwrap()) }
```

Comment:
why do sort for a empty orderedskiplist

Reply:
It's not sorting, it's presetting the sort
![image](https://github.com/cnosdb/cnosdb/assets/50371898/be222696-c109-4ab3-a424-fea526adfdcd)